web/src/components/Plugin/UI/authz-casbin.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useIntl } from "umi";
+import Form, { FormInstance } from "antd/es/form";
+import React, { useState } from "react";
+import { Radio, Input } from "antd";
+
+type Props = {
+  form: FormInstance;
+  schema: Record<string, any> | undefined;
+  ref?: any;
+  disabled?: boolean
+}
+
+const FORM_ITEM_LAYOUT = {
+  labelCol: {
+    span: 7,
+  },
+  wrapperCol: {
+    span: 8,
+  },
+};
+
+const AuthzCasbin: React.FC<Props> = ({ form, schema }) => {
+  const { formatMessage } = useIntl();
+  const properties = schema?.properties
+  const [value, setValue] = useState(1);
+  const onChange = (e: any) => {
+    setValue(e.target.value);
+  };
+
+  const [radio, setRadio] = useState('Options with path')
+  const aa = () => {
+    setRadio('Options with path')
+  }
+
+  const bb = () => {
+    setRadio('Options without path')
+  }
```

Comment:
We can put that part in the onClick, This way there is no need to define functions